src/utils.js

```diff
@@ -36,9 +36,14 @@ window.Utils = (function() {
         return tooltips;
     }
 
+    function isiOS () {
```

Comment:
This should not be necessary. You can do `AFRAME.utils.isIOS()`

Reply:
Actually I think now it's `AFRAME.utils.device.isIOS()`

Reply:
Both work